rostran/cli/__main__.py

```diff
@@ -91,7 +97,16 @@ def transform(
         elif source_path.endswith(".tf"):
             source_format = SourceTemplateFormat.Terraform
         elif source_path.endswith((".json", ".yaml", ".yml")):
-            source_format = SourceTemplateFormat.CloudFormation
+            with open(source_path, "r") as f:
+                try:
+                    tpl = json.load(f)
```

Comment:
use `yaml.load` directly due to yaml is compatible with json

Reply:
👌🏻